upstream/remote/remote.go

```diff
@@ -188,10 +196,12 @@ func (r *Remote) uploadProfile(j *upstream.UploadJob) error {
 	request.Header.Set("Content-Type", contentType)
 	// request.Header.Set("Content-Type", "binary/octet-stream+"+string(j.Format))
 
-	if r.cfg.AuthToken != "" {
+	if r.cfg.AuthToken != "" && isOGPyroscopeCloud(u) {
 		request.Header.Set("Authorization", "Bearer "+r.cfg.AuthToken)
 	} else if r.cfg.BasicAuthUser != "" && r.cfg.BasicAuthPassword != "" {
 		request.SetBasicAuth(r.cfg.BasicAuthUser, r.cfg.BasicAuthPassword)
+	} else if r.cfg.AuthToken != "" {
+		r.logger.Infof(authTokenDeprecationWarning)
```

Comment:
that's a bold move. you deprecate the usage of 'AuthToken' but in the same commit you simply removed the usage for self-hosting environments :( 
this broke our setup, downgrade was required afterwards :D